Lib/fontTools/ttLib/tables/_c_m_a_p.py

```diff
@@ -350,6 +349,10 @@ def isUnicode(self):
             self.platformID == 3 and self.platEncID in [0, 1, 10]
         )
 
+    def isUVS(self):
+        """Returns true if it is a UVS subtable"""
+        return self.platformID == 0 and self.platEncID == 5
```

Comment:
shouldn't this also check that self.format == 14? Or is platEncID=5 only used for cmap14?